### PR TITLE
✨ Don't augment root:compute

### PR DESCRIPTION
**Pull request by waltforme**
## Summary
Because we don't need the augmentations to export kube resources.

(The _REAL_ goal of this PR is to hit 1400.)

### CODE REVIEW

File: docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md [hunk 1]
```diff
@@ -21,30 +21,6 @@ IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=$SM_CONFIG kubectl kubestellar ensure
...
-apiVersion: apis.kcp.io/v1alpha1
-kind: APIBinding
-metadata:
-  name: bind-kube
-spec:
-  reference:
-    export:
-      path: "root:compute"
-      name: kubernetes
-```
```
Comment (MikeSpreitzer) [inline docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md:47]: This deletion is proper but the deleted material should be replaced by some remarks about the CRDs injected. Does not have to be done in this PR, but let's not forget to do it.

**Review by MikeSpreitzer — COMMENTED** [review]
/lgtm

**Review by ezrasilvera — APPROVED** [review]

File: docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md (hunk 1 above)
Comment (waltforme) [reply docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md:47]: Addressing this in #1417.

**Pull request merged**